bme590_melanoma_detection/validate.py

```diff
@@ -0,0 +1,28 @@
+import base64
+
+#Validate base64 image strings, ie is the string really an image?
+def is_valid_image_string(some_str):
+    try:
+        decoded_image = base64.b64decode(some_str)
+    except: 
+        return false
+    return true
+
+def check_passwords(pword1, pword2):
```

Comment:
may be a little overkill but good boilerplate for one day when you may have hashed passwords